### PR TITLE
Update Dev to Master

### PR DESCRIPTION
**Pull request by lramos15**
Sync Dev's changes into master to provide sentry logging in master and update docker file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,13 @@ A simple databaseless polling app for Slack
 
 ## How to Use
 ```
-/inorout Question
+/inorout
+Question
```
Comment (lramos15) [inline README.md:18]: This is incorrect, it is still 
```
/inorout question
Option
Option
```
unless it is multiple and/or anonymous then it's
```
/inorout [multiple] [anonymous]
question
opt1
opt2
.
.
```

File: helpFile.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+[Answer n]
+```
+
+*Simple polls*
+
+A simple poll can be created with:
+
+```
+/inorout
+Do you like DPS?
```
Comment (lramos15) [inline helpFile.md:22]: See comment in the readme

File: src/Static.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import {
+    SectionBlock, ContextBlock, PlainTextElement, Option
+} from "@slack/types";
+
+export class Static {
```
Comment (lramos15) [inline src/Static.ts]: Static is a keyword in many programming languages including Typescript. We should rename this to be something like StaticPollOperator

**Review by lramos15 — COMMENTED** [review]
This is pretty good, just some of the instructions and wording is off

File: README.md (hunk 1 above)
Comment (justetz) [reply README.md:18]: Okay, the following also works:
```
/inorout
question
opt1
opt2
.
.
```

File: src/Static.ts (hunk 3 above)
Comment (justetz) [reply src/Static.ts]: How about something more neutral, like `Helpers`?

File: README.md (hunk 1 above)
Comment (lramos15) [reply README.md:18]: Maybe for consistency we just say it's 
```
/inorout
question
opt1
opt2
.
.
.
```
an then you add the options if you want.

Comment (justetz) [reply README.md:18]: Sounds good, I'll leave the help docs as is.

**Review by justetz — APPROVED** [review]

**Pull request merged**